bundles/org.openhab.core.thing/src/main/resources/OH-INF/i18n/DefaultSystemChannels_fr.properties

```diff
@@ -16,7 +16,9 @@ channel-type.system.brightness.description = Contr
 channel-type.system.color.label = Couleur
 channel-type.system.color.description = Contr�le la couleur de la lumi�re.
 channel-type.system.color-temperature.label = Temp�rature de couleur
-channel-type.system.color-temperature.description = Contr�le la temp�rature de couleur de la lumi�re.
+channel-type.system.color-temperature.description = Contr�le la temp�rature de couleur de la lumi�re de 0 (froid) � 100 (chaud).
```

Comment:
//CC @lolodomo For approving my French skills 😉 .

Reply:
That's ok